src/frontend/citizen-portal/src/app/components/disputant-form/disputant-form.component.ts

```diff
@@ -97,7 +97,7 @@ export class DisputantFormComponent implements OnInit, AfterViewInit {
       this.driversLicenceProvinceFormControl.setValue(this.bc);
     }
 
-    if (this.preferEmail !== true) {
+    if (this.preferEmail !== undefined && this.preferEmail !== true) {
```

Comment:
Or it means `this.preferEmail !== false`

Reply:
That actually doesn't work, sorry I just tried it. On an initial OCR scan, preferEmail is undefined thus the expression "this.preferEmail != false" is true and the block is run, which is not what I want.

Reply:
Just double confirming, is it double '!==' or single '!='

Reply:
double.

Reply:
Ok, just leave it then